test/e2e/framework.go

```diff
@@ -1408,7 +1430,7 @@ func (data *TestData) deleteServiceAndWait(timeout time.Duration, name string) e
 		// Keep trying
 		return false, nil
 	}); err == wait.ErrWaitTimeout {
-		return fmt.Errorf("Service '%s' still visible to client after %v", name, timeout)
+		return fmt.Errorf("service '%s' still visible to client after %v", name, timeout)
```

Comment:
`Service` is a term so I don't think it should be changed.

Reply:
Make sense to me. Will change it back.